profiles/common/modules/features/splash_screen/splash_screen.module

```diff
@@ -70,6 +70,13 @@ function splash_screen_theme($existing, $type, $theme, $path) {
  * Implements hook_preprocess().
  */
 function splash_screen_preprocess_splash(&$variables) {
+  // Set our custom title.
+  $splash_screen_title = variable_get('splash_screen_title_value', variable_get('site_name'));
+  // Set the title.
+  drupal_set_title($splash_screen_title);
+
+  $variables['title'] = $splash_screen_title;
```

Comment:
Useless, this is the role of the "drupal_set_title" function, in ``template_preprocess_page``:
```
if (!isset($variables['title'])) {
    $variables['title'] = drupal_get_title();
  }
```